examples/buildkit0/buildkit.go

```diff
@@ -16,7 +16,7 @@ type buildOpt struct {
 
 func main() {
 	var opt buildOpt
-	flag.StringVar(&opt.target, "target", "containerd", "target (standalone, containerd)")
+	flag.StringVar(&opt.target, "target", "containerd_worker_only", "target (oci_worker_only, containerd_worker_only)")
```

Comment:
Maybe just leave `with_containerd:bool`? No need to promote these special binaries. Just switch between building containerd itself or not.